packaging/installer-linux/installer-debian/build.xml

```diff
@@ -25,10 +25,47 @@
                 </fileset>
             </copy>
 
+            <copy toFile="${project.build.outputDirectory}/@{name}/server/default/neo4j">
+                <fileset dir="${project.build.sourceDirectory}/../resources/common">
+                    <include name="default-neo4j"/>
+                </fileset>
+            </copy>
```

Comment:
Can we not do the reading of /etc/default/neo4j as a separate commit? It seems like orthogonal behaviour to me and not required for the RC.